include/mqtt/broker/retained_topic_map.hpp

```diff
@@ -358,7 +358,7 @@ class retained_topic_map {
     void dump(Output &out) {
         auto const& direct_index = map.template get<direct_index_tag>();
         for (auto const& i : direct_index) {
-            out << i.parent_id << " " << i.name << " " << (i.value ? "init" : "-") << " " << i.count << std::endl;
+            out << i.parent_id << " " << i.name << " " << (i.value ? "init" : "-") << " " << i.count << '\n';
```

Comment:
Why this replacement is needed ? Just avoid include `ostream` ?

Reply:
Correct, seeing that this is the only use of `ostream` in this file and this function is a debug function.